examples/default/main.tf

```diff
@@ -0,0 +1,8 @@
+provider "lacework" {}
+provider "aws" {
+  region = "us-west-2"
```

Comment:
Is there a reason to add the `region`?

Reply:
So that our CI works, I can add it as an environment variable.

Reply:
👍